index.yaml

```diff
@@ -1,6 +1,29 @@
 apiVersion: v1
 entries:
   agent:
+  - apiVersion: v2
+    appVersion: 3.2.0
+    created: "2023-12-19T19:15:40.686472+06:00"
+    description: Helm chart for the Optimizely Agent https://docs.developers.optimizely.com/full-stack-experimentation/docs/optimizely-agent
+    digest: 964d48b55bf52809b78bce5a81cee28fd57b432f39880c3ef31cb1f67aa1c847
+    home: https://docs.developers.optimizely.com/full-stack-experimentation/docs/optimizely-agent
+    icon: https://raw.githubusercontent.com/optimizely/optimizely-agent-helm/main/Optimizely-icon.png
+    keywords:
+    - Optimizely
+    - Optimizely Agent
+    - Full Stack
+    - Rollouts
+    - Feature Flags
+    maintainers:
+    - email: devx@optimizely.com
+      name: Optimizely Dev X Team
+    name: agent
+    sources:
+    - https://github.com/optimizely/agent
+    type: application
+    urls:
+    - https://optimizely.github.io/optimizely-agent-helm/packages/agent-1.3.0.tgz
```

Comment:
This 404-ed for me.

Reply:
Oh, this tarball is being uploaded to the repo and will be available once merged.

Heads Up: I had to add a `.gitattributes` file with `*.tgz binary` if there was any kind of checksum.

Reply:
I see helm charts repo don't need that .gitattributes, don't know the exact reason anyway. Here is a popular chart repo example https://github.com/prometheus-community/helm-charts/tree/gh-pages